### DISCUSSION TITLE
Ksf/ no access to forum

### DISCUSSION THREAD

**Discussion by japexican007** [stepmania/stepmania#2098] [Q&A]
Have piu exceed but all songs are ksf, is there a way to convert to sm without doing all the steps from scratch also don’t have access to Stepmania forums and says to go here for assistance

**Comment by Jousway** [comment]
https://github.com/stepmania/stepmania/blob/5_1-new/src/NotesLoaderKSF.cpp

should just work

**Comment by japexican007** [comment]
sorry i'm not a coder, how do i download or run this?
    On Wednesday, October 21, 2020, 10:43:05 PM PDT, Alexander Griffin <notifications@github.com> wrote:  
 
 


https://github.com/stepmania/stepmania/blob/5_1-new/src/NotesLoaderKSF.cpp

should just work

**Comment by japexican007** [comment]
right now the only way to make any song show up that has ksf is by going into edit song/steps and creating a new file via "Action" but this has me recreat all the steps from scratch, is there no easy way to just transfer the steps from ksf to sm?
    On Wednesday, October 21, 2020, 11:20:17 PM PDT, gerardo kosonoy <gerardokosonoy07@yahoo.com> wrote:  
 
  sorry i'm not a coder, how do i download or run this?
    On Wednesday, October 21, 2020, 10:43:05 PM PDT, Alexander Griffin <notifications@github.com> wrote:  
 
 


https://github.com/stepmania/stepmania/blob/5_1-new/src/NotesLoaderKSF.cpp

should just work
